backend/app/api/endpoints/openapi_responses.py

```
@@ -0,0 +1,550 @@
+    previous_task_id = None
+    if request_body.previous_response_id:
+        # Extract task_id from resp_{task_id} format
+        if request_body.previous_response_id.startswith("resp_"):
+            try:
+                previous_task_id = int(request_body.previous_response_id[5:])
+                task_id = previous_task_id  # For follow-up, use the same task_id
+            except ValueError:
+                raise HTTPException(
+                    status_code=status.HTTP_400_BAD_REQUEST,
+                    detail=f"Invalid previous_response_id format: '{request_body.previous_response_id}'",
+                )
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)